src/resources/level.py

```diff
@@ -26,19 +26,19 @@ def generate_level(self, x: int, y: int) -> None:
 
     def add_doors(self, doors: list) -> None:
         """Add doors to level"""
-        door_direction = ['right', 'bottom', 'left']
+        door_direction = ["right", "bottom", "left"]
```

Comment:
Using strings for conditionals can be error prone. It might be better to use a counter and mod it by the number of walls we want to add doors to. Ideally though we use enums.  
For example:
```python
while doors:
   direction = doors % 3   # assuming three walls
   doors -= 1
```

Reply:
Fixed (we don't want it to be spread out equally though hence I did ``randint(0, doors)`` in order to get random door placement